packages/form-field/src/FormFieldInput/FormFieldInput.types.ts

```diff
@@ -0,0 +1,8 @@
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+
+import { FormFieldChildren } from '../FormField/FormField.types';
+
+export interface FormFieldInputProps extends HTMLElementProps<'div'> {
```

Comment:
Should extend HTMLElementProps<'input'>

Reply:
No since it's the wrapper around the input (this bit)
![Screenshot 2023-10-05 at 1 26 49 PM](https://github.com/mongodb/leafygreen-ui/assets/2414030/e5a509fe-30e3-472f-ad09-0ab42461e2cd)

Reply:
I think it should be one of those two though, otherwise it's a bit confusing and I'd expect the component to return an input itself

Reply:
the only other option would be to omit the component entirely - render a div that handles these styles 
```
<div className={wrapperStyles}>{children}</div>
```

and then expose a prop to pass the style overrides there (which we discussed previously, and were in favor of the more explicit)

Reply:
I _think_ I prefer `...Container` vs `...Wrapper`. Thoughts?